.pipelines/build-linux.sh

```diff
@@ -16,7 +16,7 @@ make
 cd $DIR
 
 echo "------------ Bundle Shell Extension Scripts for Agent Release -------------------------"
-cd $DIR/../deployment/agent-deployment/ServiceGroupRoot/Scripts
+cd $DIR/../deployment/mergebranch-multiarch-agent-deployment/ServiceGroupRoot/Scripts
```

Comment:
Not a 100% sure if we even use this file.

Reply:
yes not sure neither, maybe keep it for now, will check with Ganga

Reply:
@wanlonghenry , lets sync up on this tomorrow morning. I see you have deleted "pushAgentToAcr.sh" which is being deleted in this PR. We need to make sure this doesnt cause any regressions.